include/dxc/dxcapi.h

```diff
@@ -569,7 +569,12 @@ struct IDxcVersionInfo : public IUnknown {
 
 CROSS_PLATFORM_UUIDOF(IDxcVersionInfo2, "fb6904c4-42f0-4b62-9c46-983af7da7c83")
 struct IDxcVersionInfo2 : public IDxcVersionInfo {
-  virtual HRESULT STDMETHODCALLTYPE GetCommitInfo(_Out_ UINT32 *pCommitCount, _Out_ char **pCommitHash) = 0;
+  virtual HRESULT STDMETHODCALLTYPE GetCommitInfo(_Out_ UINT32 *pCommitCount, _Outptr_result_z_ char **pCommitHash) = 0;
+};
+
+CROSS_PLATFORM_UUIDOF(IDxcVersionInfo3, "5e13e843-9d25-473c-9ad2-03b2d0b44b1e")
+struct IDxcVersionInfo3 : public IDxcVersionInfo2 {
+  virtual HRESULT STDMETHODCALLTYPE GetCustomVersionString(_Outptr_result_z_ char **pCommitHash) = 0;
```

Comment:
Shouldn't this be named something other than `pCommitHash`?

Reply:
Also we don't communicate the protocol here that this is a CoTaskMemAlloc that must be freed by the user with CoTaskMemFree.  I know this is an existing problem with the previous interface method as well.  Is there a SAL annotation that can be used to express this?  Or at least a comment?  We have the same issue with IDxcCompiler2::Compile2(..., ppDebugBlobName, ...), but it says `(Must be HeapFree()'d!)`, which isn't accurate it would seem.

Reply:
Fixed typo, added a comment saying it must be HeapFree()'d